index.html

```diff
@@ -1,54 +1,378 @@
-  <title>Zunair Mobile Setup Website</title>
-</head>
-
-<body>
-  <div class="container">
-    <header class="header-section">
-      <nav class="nav-bar">
-        <a href="#" class="logo">Zunair khan</a>
-        <i class="fa fa-bars menu-bar" aria-hidden="true"></i>
-      </nav>
-    </header>
-    <section class="headline-section">
-      <div class="headline-container">
-        <div class="heading-1">
-          <h1>Hey There. I'm Zunair</h1>
-        </div>
-        <div class="heading-2">
-          <h2>I'am a software developer</h2>
-        </div>
-        <div class="paragraph">
-          <p>
-            I can help you build a product , feature or website Look through
-            some of my work and experience! If you like what you see and have
-            a project you need coded,don’t hestiate to contact me.
-          </p>
-        </div>
-        <div class="social-icons-container">
+  <body>
+    <div class="container">
+      <header class="header-section">
+        <nav class="nav-bar">
+          <a href="#" class="logo">Zunair khan</a>
+          <i class="fa fa-bars menu-bar" aria-hidden="true"></i>
+        </nav>
+      </header>
+      <section class="headline-section">
+        <div class="headline-container">
+          <div class="heading-1">
+            <h1>Hey There. I'm Zunair</h1>
+          </div>
+          <div class="heading-2">
+            <h2>I'am a software developer</h2>
+          </div>
+          <div class="paragraph">
+            <p>
+              I can help you build a product , feature or website Look through
+              some of my work and experience! If you like what you see and have
+              a project you need coded,don’t hestiate to contact me.
+            </p>
+          </div>
+          <div class="social-icons-container">
+            <ul>
+              <li>
+                <a href="#"
+                  ><i class="fa fa-github social-icons" aria-hidden="true"></i
+                ></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><i class="fa fa-linkedin social-icons" aria-hidden="true"></i
+                ></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><i
+                    class="fa fa-angellist social-icons"
+                    aria-hidden="true"
+                  ></i
+                ></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><i class="fa fa-twitter social-icons" aria-hidden="true"></i
+                ></a>
+              </li>
+              <li>
+                <a href="#" class="vectorimg"
+                  ><img src="/images/Vector.png" class="vectorimg" alt="Vector"
+                /></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </section>
+
+      
+
+      <section class="work-section">
+        <div class="box-1">
+          <h1>My Recent Works</h1>
+        </div>
```

Comment:
[OPTIONAL} - H1 tags are the main header in a web article. There should only be one H1 per page. 

But since you have more than one h1 tag on a page, please kindly change the other h1 tags to an h2 or h3. Your h1 tag should be at the top of the page content (above any other heading tags in the page code) 👌